src/NuGet.Clients/NuGet.PackageManagement.PowerShellCmdlets/Cmdlets/UpdatePackageCommand.cs

```diff
@@ -150,6 +153,19 @@ protected override void ProcessRecordCore()
             TelemetryService.EmitTelemetryEvent(actionTelemetryEvent);
         }
 
+        protected override void ValidateArgumentsAreSupported()
```

Comment:
this isn't as much as validating as it is warning. i suggest change name to something like : WarnForPackageReferenceProject